regen-network/regen-web
# feat: APP-9 Add a banner at top of project pages and individual profile page on first view

## Description

https://regennetwork.atlassian.net/jira/software/c/projects/ENG/boards/51?selectedIssue=APP-9



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



- BannerCard component on storybook: https://deploy-preview-2336--regen-storybook.netlify.app/?path=/story/molecules-bannercard--default

- From https://deploy-preview-2336--regen-marketplace.netlify.app/:
1. Login
2. Visit your own profile page, you should see the banner at the top. If you navigate away and navigate back to this page, it won't be there anymore
3. Visit a few of your project pages, you should see the project banner at the top when first viewing a given project page

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
